packages/@aws-cdk/aws-ecs-patterns/lib/fargate/application-load-balanced-fargate-service.ts

```diff
@@ -92,6 +92,13 @@ export interface ApplicationLoadBalancedFargateServiceProps extends ApplicationL
    * @default - A new security group is created.
    */
   readonly securityGroups?: ISecurityGroup[];
+
+  /**
+   * The runtime platform of the task definition
+   *
+   * @default - If the property is undefined, `operatingSystemFamily` is LINUX and `cpuArchitecture` is X86_64
```

Comment:
```suggestion
   * @default - `operatingSystemFamily` is LINUX and `cpuArchitecture` is X86_64
```

same for the others.